server2/Arcadia.Assistant/Arcadia.Assistant.PushNotifications.Contracts/Models/PushNotificationContent.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Arcadia.Assistant.PushNotifications.Contracts.Models
+{
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    [KnownType(typeof(IReadOnlyDictionary<string, string>))]
```

Comment:
why do you need KnownTypeAttribute here?

Reply:
Changed for correct type